### PR TITLE
Add Dot1AD frame

### PR DESCRIPTION
**Pull request by teodoro-vargas**

### CODE REVIEW

File: tests/src/dot1q_test.cpp [hunk 1]
```diff
@@ -82,18 +82,6 @@ TEST_F(Dot1QTest, Id) {
     EXPECT_EQ(3543, dot1.id());
 }
 
-TEST_F(Dot1QTest, QinQ) {
```
Comment (dajose) [inline tests/src/dot1q_test.cpp:85]: why did you remove this one?

Comment (teodoro-vargas) [reply tests/src/dot1q_test.cpp:85]: Because the original code relied in the `Dot1Q` frame to represent both the protocols Dot1Q and Dot1AD. This caused issues when the packets  were rendered: the first frame would have PID 0x8100 (Dot1Q) while the second frame would have PID 0x88a8 (Dot1AD). The correct sequence is with these values inverted: https://upload.wikimedia.org/wikipedia/commons/thumb/5/52/TCPIP_802.1ad_DoubleTag.svg/1280px-TCPIP_802.1ad_DoubleTag.svg.png

Comment (teodoro-vargas) [reply tests/src/dot1q_test.cpp:85]: You actually made me realize I missed something. I'm going to make another change.

**Comment by teodoro-vargas** [comment]
PR updated :)

**Comment by dajose** [comment]
@teodoro-vargas remove style changes (like white-spaces and newlines)

**Comment by teodoro-vargas** [comment]
@dajose :(
My text editor removes trailing whitespaces by default.

**Comment by teodoro-vargas** [comment]
Friendly ping 👍

**Comment by dajose** [comment]
seems good to me, let's wait @carlos-jenkins
